rpcserver.go

```diff
@@ -3476,10 +3476,17 @@ func (r *rpcServer) fetchPendingOpenChannels() (pendingOpenChannels, error) {
 		// broadcast.
 		// TODO(roasbeef): query for funding tx from wallet, display
 		// that also?
+		var witnessWeight int64
```

Comment:
Since we didn't catch any of these three code paths in the integration tests, do you think it would be easy to add such a test (or tests)?

Reply:
Added an itest